backend/backend.dockerfile

```diff
@@ -1,29 +1,61 @@
-FROM python:3.7
+FROM python:3.8-slim as base
 
-LABEL maintainer="OpenStax Content Engineering"
+RUN apt-get update -qq \
+ && apt-get install -y --no-install-recommends \
+    # required by psycopg2 at build and runtime
+    libpq-dev \
+    curl \
+ && apt-get autoremove -y
```

Comment:
Also both ampersand lines could use one additional space of indentation if formatting is to be consistent.